protocol/invocation/rpcinvocation.go

```diff
@@ -141,6 +143,29 @@ func (r *RPCInvocation) SetCallBack(c interface{}) {
 	r.callBack = c
 }
 
+func (r *RPCInvocation) ServiceKey() string {
+	intf := r.AttachmentsByKey(constant.INTERFACE_KEY, "")
+	if intf == "" {
```

Comment:
hey guy, in dubbo-go, we check a string is nil or not by `len(string) == 0` instead of `str == ""`.

Reply:
Fixed. Thanks